src/components/Layout/Header/Header.jsx

```diff
@@ -16,13 +18,13 @@ export function Header() {
 		<header className={styles.header}>
 			<div className="__container">
 				<div className={styles.logo}>
-					Proved<span>Code</span>
+					<Link to="/">Proved<span>Code</span></Link>
 				</div>
 				<nav className={styles.nav}>
 					{menuItems.map(({ title, link }, index) => (
-						<a href={link} key={index}>
+						<Link to={link} key={index}>
```

Comment:
you`d better use NavLink tag, it will help you to control which link is active at current time
```suggestion
						<NavLink to={link} key={index}>
```